components/inputs/input-checkbox.js

```diff
@@ -276,6 +276,10 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 		this._isHovered = false;
 	}
 
+	#handleSupportingChange(e) {
```

Comment:
Are there any other events we should be stopping?

Reply:
`change` is the only event that checkbox itself fires, which is why it's getting mixed up here. Like technically they could have wired up to `click`, `mouseover`, etc. but IMO clicking in the supporting slot should probably dispatch a click on the whole component anyway.